### PR TITLE
Object store integration

### PR DESCRIPTION
**Pull request by tahsin352**
https://git.civo.com/documentation/rfcs/-/tree/main/2022-01-28-object-storage

### CODE REVIEW

File: app/models/civo/object_store.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module Civo
+  class ObjectStore < Base
+    get :all, "/v2/objectstores", requires: [:region]
+    get :find, "/v2/objectstores/:id", requires: [:id]
+    post :create, "/v2/objectstores", requires: [:name, :max_size_gb, :max_objects]
+    put :update, "/v2/objectstores/:id", requires: [:id]
+    delete :remove, "/v2/objectstores/:id", requires: [:id]
```
[the remark this answers is not in the document]
Comment (andyjeffries) [reply app/models/civo/object_store.rb:7]: Also, why is `max_size_gb` and `max_objects` required, they should have default values. Also, I thought `max_objects` was going away.

[the remark this answers is not in the document]
Comment (kaihoffman) [reply app/models/civo/object_store.rb:7]: Agreed, `region` is required pretty much on all other calls of this type for the other resources, would make sense to be required on these.

[the remark this answers is not in the document]
Comment (kaihoffman) [reply app/models/civo/object_store.rb]: So @andyjeffries - should the `max_size_gb` provide a default value here?

[the remark this answers is not in the document]
Comment (andyjeffries) [reply app/models/civo/object_store.rb]: I think so. 500GB is the minimum and it's the version likely everyone will choose.

[the remark this answers is not in the document]
Comment (tahsin352) [reply app/models/civo/object_store.rb]: Updated.

**Pull request merged**